src/IO/Observer/Helpers.hpp

```diff
@@ -27,6 +29,18 @@ struct make_reduction_data_tag_impl {
   using type = tmpl::wrap<typename ReductionDataType::datum_list,
                           ::observers::Tags::ReductionData>;
 };
+
+template <typename Metavariables>
+std::string input_source_from_cache(
```

Comment:
I don't think this should be a `detail`.

Reply:
I will move out of the detail namespace